lib/matplotlib/axes/_axes.py

```diff
@@ -3095,7 +3095,7 @@ def boxplot(self, x, notch=None, sym=None, vert=None, whis=None,
                 showbox=None, showfliers=None, boxprops=None,
                 labels=None, flierprops=None, medianprops=None,
                 meanprops=None, capprops=None, whiskerprops=None,
-                manage_xticks=True, autorange=False):
+                manage_xticks=True, autorange=False, zorder=2):
```

Comment:
Good idea. I debated doing it this way, but `streamplot` had a default zorder of 2 IIRC, so I went with that convention.

I'll make the change in a bit.